### PR TITLE
set-up-coin-info-component

### PR DESCRIPTION
**Pull request by jvin08**

### CODE REVIEW

**Review by Basiol97 — APPROVED** [review]

File: app/components/coinPage/MainInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import Image from "next/image";
+import { Triangle, Squares, SmallTriangle } from "./SVGComponents";
+import Link from "next/link";
+import { addCommas, isPositive } from "./utils";
+type Coin = {
+  image: {large: string},
+  name: string,
+  symbol: string,
+  links: {homepage: string[]}
+  market_data:any
```
Comment (Basiol97) [inline app/components/coinPage/MainInfo.tsx]: why any

File: app/components/coinPage/MainInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  isStorageInfo,
+  maxPriceRange,
+  rank,
+  currencyLabel
+}:{
+  data: Coin, 
+  currencySign:string, 
+  gainOrLoss:string,
+  isStorageInfo:boolean,
+  maxPriceRange:{prices: any},
```
Comment (Basiol97) [inline app/components/coinPage/MainInfo.tsx]: any

File: app/components/coinPage/MainInfo.tsx (hunk 1 above)
Comment (jvin08) [reply app/components/coinPage/MainInfo.tsx]: I agree, will fix next commit

File: app/components/coinPage/MainInfo.tsx (hunk 2 above)
Comment (jvin08) [reply app/components/coinPage/MainInfo.tsx]: agreed

**Pull request merged**